SPECS/binutils/binutils.spec

```diff
@@ -20,14 +20,14 @@
 
 Summary:        Contains a linker, an assembler, and other tools
 Name:           binutils
-Version:        2.41
-Release:        6%{?dist}
+Version:        2.44
+Release:        1%{?dist}
 License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Base
 URL:            https://www.gnu.org/software/binutils
-Source0:        https://ftp.gnu.org/gnu/binutils/%{name}-%{version}.tar.xz
+Source0:        https://ftp.gnu.org/gnu/binutils/%{name}-with-gold-%{version}.tar.xz#/%{name}-%{version}.tar.xz
```

Comment:
Why is this file being renamed from `binutils-with-gold-2.44.tar.xz` to `binutils-2.44.tar.xz`?

Reply:
I thought we were meant to name all source files in that format, that is all. I can change it to use the default name if you prefer.